pkg/config/config.go

```diff
@@ -66,7 +66,7 @@ func init() {
 	pflag.Uint64("max-content-length", 5*1024*1024, "Max content size in bytes - requests with body bigger than this value will be skipped")
 	pflag.Bool("enable-animated-thumbnails", true, "When enabled, will attempt to use libvips library to build animated thumbnails. Can increase CPU usage and cache storage by a lot. Enabled by default")
 	pflag.Uint("max-thumbnail-size", 300, "Maximum width/height pixel size count of the thumbnails sent to the clients.")
-	pflag.Duration("twitch-username-cache-duration", 1*time.Hour, "Cache timeout for twitch usernames")
+	pflag.Duration("twitch-username-cache-duration", 10*time.Minute, "Cache timeout for twitch usernames")
```

Comment:
Document this in the readme

Reply:
I changed the deefault value in `config.yaml`, not sure where to place it in the README

Reply:
oops I meant the changelog - added in [`3f4730c` (#714)](https://github.com/Chatterino/api/pull/714/commits/3f4730c97399b2c2ad6836350e17da1d95ed4513)